pkg/ingress/gateway_test.go

```diff
@@ -341,20 +156,17 @@ func TestHandleCertificateChange(t *testing.T) {
 				msgBroker:    msgBroker,
 			}
 
-			go c.handleCertificateChange(tc.previousCertSpec, tc.stopChan)
-			defer close(tc.stopChan)
+			go c.provisionIngressGatewayCert(tc.previousCertSpec, stop)
```

Comment:
Could you explain why the `defer close(tc.stop)` is unnecessary here?

Reply:
i took off stop from the test case (it was never really a variable, just always the same), and reuse the stop chan from line 135, which already has a deferred close on it.

Reply:
Got it! Missed the the earlier defer close(stop). Thanks!